packages/dd-trace/src/appsec/reporter.js

```diff
@@ -47,6 +45,14 @@ const IDENTIFICATION_HEADERS_MAP = mapHeaderAndTags([
   'x-sigsci-requestid',
   'x-sigsci-tags',
   'akamai-user-risk'
+]
+
+// these request headers are always collected - it breaks the expected spec orders
```

Comment:
> it breaks the expected spec orders

This comment is added because the comment in line 21 says:

> following header lists are ordered in the same way the spec orders them, it doesn't matter but it's easier to compare

Reply:
Almost, but `content-type`, `user-agent` and `accept` aren't in their correct positions.

Reply:
Ah yes I see, they are in order if you filter by "asm enabled" only but anyway, lgtm